background.ts

```diff
@@ -388,7 +388,7 @@ ipcMainProxy.on('settings-read', (event) => {
 // This is the synchronous version of the above; we still use
 // ipcRenderer.sendSync in some places, so it's required for now.
 ipcMainProxy.on('settings-read', (event) => {
-  console.debug(`event settings-read in main: ${ event }`);
+  console.debug(`event settings-read in main: ${ JSON.stringify(cfg) }`);
```

Comment:
Isn't this unnecessary now, given that `ipcMainProxy` automatically logs `settings-read` events?

Reply:
I kept this one because it logs the details of the cfg object that the event handler doesn't have access to.